### ISSUE TITLE
Can't set the pad on Motorola Defy

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ionux/genplus-droid#6]
```
What steps will reproduce the problem?
1. Open GensPlusDroid
2. Settings
3. Try to configure the keys

What is the expected output? What do you see instead?
I've expected to see the pads on the screen, but the emulator didn't show it.

What version of the product are you using? On what operating system?
GensPlusDroid 1.7
Android 2.2
Motorola Defy

Please provide any additional information below.
Wuthout the pad i cannot play. =/


```

Original issue reported on code.google.com by `claudiob...@gmail.com` on 8 Sep 2011 at 2:22

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in version 1.8, please verify.
```

Original comment by `shinhals...@gmail.com` on 28 Sep 2011 at 5:02
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
With the 1.8 version it works perfectly! Thank you very much!
```

Original comment by `claudiob...@gmail.com` on 28 Sep 2011 at 12:49
